src/content/getting-started/cluster-size-autoscaling/index.md

```diff
@@ -85,7 +85,7 @@ In workload clusters without autoscaling support, the number of Ingress Controll
 ## See also
 
 - [Cluster Autoscaler advanced configuration]({{< relref "/advanced/cluster-autoscaler" >}})
-- [Recommendations and Best Practices regarding cluster size]({{< relref "/getting-started/best-practices#cluster-sizing" >}})
+- [Recommendations and Best Practices regarding cluster size]({{< relref "/support/best-practices#cluster-sizing" >}})
```

Comment:
Best practices shouldn't be under support imo. They are better within the hands-on guides. "getting-started" sounds good to me.

Reply:
hmm yeah I don't have a strong feeling, I moving after a discussion with Carlos. I see there we could have content which is not only for starters, but I see it in both sections